scipy/_lib/_array_api.py

```diff
@@ -679,7 +679,7 @@ def _make_sphinx_capabilities(
     return capabilities
 
 
-def _make_capabilities_note(fun_name, capabilities):
+def _make_capabilities_note(fun_name, capabilities, extra_note=None):
```

Comment:
Nice. I've wanted something like this before but didn't think to just add it here. I'll use this in my next `signal` PR.